celestia.go

```diff
@@ -0,0 +1,101 @@
+package celestia
+
+import (
+	"context"
+	"time"
+
+	openrpc "github.com/rollkit/celestia-openrpc"
```

Comment:
We should not have to import celestia-openrpc , We want to import celestia-node directly.